features/auth/login/LoginForm.tsx

```diff
@@ -129,7 +129,7 @@ export default function LoginForm() {
         <div className={classes.loginOptions}>
           <FormControlLabel
             className={classes.rememberSwitch}
-            control={<Switch size="small" />}
+            control={<Switch size="small" checked={true} />}
```

Comment:
I'm not sure this actually ever worked 🤔 did a bit of code digging and it looks like the frontend "forgot" to pass the value of this properly to the backend on the API call...

Reply:
what if the remember me switch creates a cookie and at the start of the page check if that cookie exists? I guess that could be a good way to do it? (without needing to sent this value to the server, but I don't know if this is something you intentionally want...)

Reply:
if the pseudo code it's clear I can implement it :)

Reply:
If you look at the [proto definition of the auth req](https://github.com/Couchers-org/couchers/blob/4e87f4c0cd2e72f3969b86b7de32e875fccbeafb/app/proto/auth.proto#L203-L206), I believe it is important we send this back to the server/backend so the session remains valid for longer.

@aapeliv may have more details of what `the session is valid from creation until expiry, with no need to be used in between` means (e.g. I'm not sure how expiry is determined)

Reply:
High level outline of approach:
- Make sure the switch has a `name` property as that's important for React hook Form to update the form state correctly
- Update `authActions.passwordLogin` to accept the `rememberMe` boolean
- Check that the underlying call in that function to the gRPC API also takes this value in - you may have to update some code in one of the function in the `services` folder
- Test that it works? Actually this is where I'm a bit confused 😅 , as even the state of it right now, you don't have to log in again anyway? So is remember me just means you don't have to type in your email/password again?